tools/clang/test/HLSL/workgraph/work-graphs.hlsl

```diff
@@ -308,3 +308,24 @@ void node4_06(RWThreadNodeInputRecord<RECORD> input)
 {
   input.FinishedCrossGroupSharing(); // expected-error {{no member named 'FinishedCrossGroupSharing' in 'RWThreadNodeInputRecord<RECORD>'}}
 }
+
+struct R
+{
+  uint3 dtid: SV_DispatchThreadID;
+  uint3 gid: SV_GroupID;
+  uint gidx: SV_GroupIndex;
+  uint3 gtid: SV_GroupThreadID;
+};
+
+[Shader("node")]
+[NodeLaunch("coalescing")]
```

Comment:
I just realized another problem with this shader: `"coalescing"` launch only supports `SV_GroupIndex` and `SV_GroupThreadID`.  This must be changed to `"broadcasting"`, otherwise diagnostics that we should be implementing would cause this to fail.  I've filed #5827 to track diagnostics and validation implementation for these.

Reply:
And note: changing this to `"broadcasting"` will require a change of `RWGroupNodeInputRecords` to `RWDispatchNodeInputRecord`, and the addition of a `[NodeDispatchGrid(...)]` attribute, since you wouldn't want to add a `SV_DipatchGrid` field to the record, or you wouldn't be able to use the same struct in both places.

Reply:
switched to broadcasting.

Reply:
It looks like you missed this file in particular.

Reply:
Fixed.